module/proto/peggy/v1/msgs.proto

```diff
@@ -137,6 +140,24 @@ message MsgConfirmBatch {
 
 message MsgConfirmBatchResponse {}
 
+// MsgConfirmLogicCall
```

Comment:
Comment seems to be for something else, not this code

Reply:
copy pasta 🙈 =